Makefile

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 #
 
+VERSION := 0.0.1
```

Comment:
Can we still set the version to `$(date '+%Y.%m.%d.%H')` by default?

It's probably also worth adding a comment to explain that we do it this way so an external build system can easily configure the version. Otherwise it'd be hard to know why we don't just use the string "0.0.1" below in the "dch" command.